src/devices/Bmp180/Bmp180.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>    
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Bmp180.cs" />
+    <Compile Include="CalibrationData.cs" />
+    <Compile Include="Register.cs" />
+    <Compile Include="Sampling.cs" />
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease.*" />

```

Comment:
We have recently converted all projects to use `$(SystemDeviceGpioPackageVersion)` instead of version, would be nice if this was converted too

Reply:
done